### PR TITLE
remove unet

### PR DESCRIPTION
**Pull request by JohnCorby**
removes UnityEngine.Networking.dll to make everything qnet

also has 1-line fix for a _baseIsReady bug

### CODE REVIEW

File: QSB/Syncs/SyncBase.cs [hunk 1]
```diff
@@ -73,7 +73,7 @@ public uint PlayerId
 		public PlayerInfo Player => QSBPlayerManager.GetPlayer(PlayerId);
 		private bool _baseIsReady => QSBPlayerManager.PlayerExists(PlayerId)
 			&& Player != null
-			&& Player.IsReady
+			// && Player.IsReady
```
Comment (misternebula) [inline QSB/Syncs/SyncBase.cs:76]: ?

File: QuantumUNET/QuantumUNET.csproj [hunk 2]
```diff
@@ -15,9 +15,5 @@
   <ItemGroup>
     <PackageReference Include="OuterWildsGameLibs" Version="*" IncludeAssets="compile" />
     <Reference Include="System.Data.DataSetExtensions" />
-    <Reference Include="UnityEngine.Networking, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\QSB\lib\UnityEngine.Networking.dll</HintPath>
-      <SpecificVersion>False</SpecificVersion>
-    </Reference>
```
Comment (misternebula) [inline QuantumUNET/QuantumUNET.csproj:21]: UnityEngine.Networking is used in QNetworkServer though

File: QSB/Syncs/SyncBase.cs (hunk 1 above)
Comment (JohnCorby) [reply QSB/Syncs/SyncBase.cs:76]: this is the _baseIsReady bug fix

File: QuantumUNET/QuantumUNET.csproj (hunk 2 above)
Comment (JohnCorby) [reply QuantumUNET/QuantumUNET.csproj:21]: not anymore it isnt. the whole point is that a moved everything used to qnet so we dont need the dep anymore

**Review by misternebula — APPROVED** [review]

**Pull request merged**